extensions/eclipse-che-theia-user-preferences/package.json

```diff
@@ -12,14 +12,14 @@
     "src"
   ],
   "dependencies": {
-    "@theia/core": "next",
+    "@theia/core": "0.11.0",
     "nsfw": "^1.2.2",
-    "@theia/preferences": "next",
+    "@theia/preferences": "0.11.0",
     "@eclipse-che/theia-plugin-ext": "^0.0.1"
   },
   "devDependencies": {
     "rimraf": "latest",
-    "typescript": "latest"
+    "typescript": "3.1.3"
```

Comment:
@benoitf could you please explain why do we need to fix typescript version here?

Reply:
@mmorhun because it's tagged everywhere else. I think it's a mistake to have latest there

Reply:
https://github.com/eclipse/che-theia/blob/master/package.json#L13

yes, it was done on purpose. I had invalid typescript dependencies.

we could cleanup later dependencies as it seems a lot of dependencies are in main package.json